src/main_helpers.js

```diff
@@ -47,6 +47,9 @@ export function findInvestBinaries(isDevMode) {
     }
     logger.info(`Found invest binaries ${investExe} for version ${investVersion}`);
     resolve([investExe, `${investVersion}`.trim(os.EOL)]);
+  }).catch(error => {
+    console.log(error.message);
+    console.log('InVEST binaries are probably missing.');
```

Comment:
I'm curious about the intent here. There's an earlier try/catch that does a similar thing (though maybe it's message could be improved). And the earlier `catch` also `throws`, which would now get caught here. If it's useful to catch that error, then maybe we simply don't want to `throw` it on line 46? Or are there other errors getting caught here that miss the earlier try/catch?

Also, I'd prefer a `logger.error` instead of `console.log`. Then it goes to the logfile and the console (when in dev mode).

Reply:
This is a good catch Dave and comment about using `logger`.

Reply:
The main intent here is to avoid the ugly `UnhandledPromiseRejectionWarning` which you currently get if something goes wrong in this function. Per [this stackoverflow post](https://stackoverflow.com/questions/40500490/what-is-an-unhandled-promise-rejection), every `Promise` is expected to have a `.catch()` that handles Promise rejection. So we're getting a Warning on top of an error, because we're not handling the error in any way. I hadn't noticed the earlier `try/catch` but I agree it doesn't make sense to have both. I don't think there's any need for the first one. 
Will replace `console.log` with `logger.error` :)